opentelemetry-sdk/src/metrics/meter_provider.rs

```diff
@@ -113,6 +114,16 @@ impl SdkMeterProvider {
     }
 }
 
+impl Drop for SdkMeterProvider {
+    fn drop(&mut self) {
+        if self.is_shutdown.load(Ordering::Relaxed) {
+            return;
+        }
+        if let Err(err) = self.shutdown() {
```

Comment:
we need to set is_shutdown to true, right?

Reply:
nvm

Reply:
i think its best to move the check of is already shutdown to inside the shutdown() method itself, so drop need not worry about that.

Reply:
+1 on that. This doesn't seem to fit Drop's job of `clean up resource`

Reply:
Good point. Remove the chek from `drop`. shutdown() is already validating it.